src/main/java/com/ericsson/ei/handlers/ObjectHandler.java

```diff
@@ -278,9 +282,9 @@ public String lockDocument(String id) {
      */
     public int getTtl() {
         int ttl = 0;
-        if (ttlValue != null && !ttlValue.isEmpty()) {
+        if (!StringUtils.isEmpty(aggregationsTtl)) {
```

Comment:
Why not use `StringUtils.isNotEmpty(aggreationsTtl)`? https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNotEmpty-java.lang.CharSequence-

Reply:
True, it's more readable - I'll change to that instead.

Reply:
I didn't know it also had a isNotEmpty method. =)